README.adoc

```diff
@@ -12,7 +16,9 @@ copied are automatically
 https://wiki.jenkins.io/display/JENKINS/Fingerprint[fingerprinted] for
```

Comment:
There looks no alternate documentation in https://jenkins.io/

Reply:
Created https://github.com/jenkins-infra/jenkins.io/issues/2908

Reply:
Thank you.